pkg/interfaces/types.go

```diff
@@ -12,21 +12,22 @@ const (
 	StackLabel = "com.docker.stacks.stack_id"
 )
 
-// SwarmStack represents a Stack with all of its elements converted to Engine
+// Stack represents a Stack with all of its elements converted to Engine
 // API types.
-// NOTE: SwarmStack is only used internally for reconciliation of Swarm
+// NOTE: Stack is only used internally for reconciliation of Swarm
 // stacks and is never exposed via the API.
-type SwarmStack struct {
-	ID   string
-	Meta swarm.Meta
-	Spec SwarmStackSpec
+type Stack struct {
+	Version swarm.Version
```

Comment:
Version is not needed here -- it should be part of Meta

Reply:
fixed yet moved to pkg/types/stack.go